### DISCUSSION TITLE
Différences au niveau de l'indexation elasticsearch suite aux supprésions de données (métallurgie)

### DISCUSSION THREAD

**Discussion by romain-b13** [SocialGouv/code-du-travail-numerique#5532] [Technique]
### Articles liés

**Preprod**  : 

![Image](https://github.com/SocialGouv/code-du-travail-numerique/assets/84397271/e6c0f5c9-9e32-405a-9ee7-4c1e400d50b7)

**Prod** : 

![Image](https://github.com/SocialGouv/code-du-travail-numerique/assets/84397271/be9d1106-f1a8-4863-a587-ffc21bc59314)

### Moteur de recherche 

**Preprod** : 

![Image](https://github.com/SocialGouv/code-du-travail-numerique/assets/84397271/552cd8dd-7b4f-4f59-a2c3-8f9853b33b79)

**Prod** : 

![Image](https://github.com/SocialGouv/code-du-travail-numerique/assets/84397271/b571cd98-76cd-4928-ba2a-321a37d46003)

Ecarts observées aussi pour les requêtes suivantes :

- A quelle date l’employeur doit-il me remettre mes documents de fin de contrat ?
- A quelle date l’employeur doit-il régler mon salaire ?

L'écart ne se produit pas sur les requêtes préqualifiés

**Comment by maxgfr** [comment]
**Pour la recherche** 

J'ai regardé si nous avons fait des itérations sur la partie frontend, et si celle-ci posait soucis.

Il semble que non. Le problème ne provient du frontend, ni du backend.

Entre la version en production et celle de la preproduction, le seul paramètre ayant changé est le nombre de contributions (de 4500 à 2000 environ), et le nombre de convention collective (de 700 à 600 environ)

Ma supposition me pousse à dire que les résultats ont changé suite à la diminution du nombre de documents ingérés

Pour information, aucun snapshot a bougé sur la recherche au niveau de l'API.

**Pour les articles liés** 

Les articles liés sont générés à partir des covisites et de la recherche.

Concernant les covisites, elles ne sont pas fonctionnelles.

En sachant que nous avons la partie recherche qui a bougé, le fait que les articles liés bougent également prouvent que le soucis provient de la donnée et non des requêtes.

Ce soucis touche tout type de contenu et non exclusivement les contributions et les conventions collectives
